lib/quby/questionnaires/entities/questionnaire.rb

```diff
@@ -183,9 +181,20 @@ def to_codebook(options = {})
             output << ""
           end
 
+          flags.each do |_flag_key, flag|
```

Comment:
Het is geen snelheidsding, flatten is zelfs net iets sneller, maar volledig verwaarloosbaar. 
Het is idd jammer van de `.map { |s| s + "\n" }`, zonder dat was het perfect geweest, in 1x een high-level overzicht van wat er in komt. 
Wat mij betreft 10x leesbaarder/scanbaarder dan de appending methode en veel meer railsy. 
Maar wil best toegeven dat dat een mening is en dat andere aanpakken ook best leesbaar kunnen zijn.